src/main/java/io/th0rgal/oraxen/mechanics/provided/gameplay/stringblock/StringBlockMechanicListener.java

```diff
@@ -170,42 +178,32 @@ public void onBreakingCustomBlock(final BlockBreakEvent event) {
         final Block blockAbove = block.getRelative(BlockFace.UP);
         final Player player = event.getPlayer();
         for (BlockFace face : BlockFace.values()) {
-            if (face == BlockFace.SELF && !face.isCartesian()) continue;
+            if (face == BlockFace.SELF) continue;
             if (block.getType() == Material.TRIPWIRE || block.getType() == Material.NOTE_BLOCK) break;
             if (block.getRelative(face).getType() == Material.TRIPWIRE) {
-                StringBlockMechanic mechanic = getStringMechanic(block.getRelative(face));
```

Comment:
1. Event used to be wrongly triggered
2. Need further check if remove mechanic check would cause side effects